internal/lm/nvml.go

```diff
@@ -101,11 +101,21 @@ func newVersionLabeler(manager resource.Manager) (Labeler, error) {
 	}
 
 	labels := Labels{
+		// Deprecated labels
 		"nvidia.com/cuda.driver.major":  driverMajor,
 		"nvidia.com/cuda.driver.minor":  driverMinor,
 		"nvidia.com/cuda.driver.rev":    driverRev,
 		"nvidia.com/cuda.runtime.major": fmt.Sprintf("%d", *cudaMajor),
 		"nvidia.com/cuda.runtime.minor": fmt.Sprintf("%d", *cudaMinor),
+
+		// New labels
+		"nvidia.com/cuda.driver-version.major":    driverMajor,
+		"nvidia.com/cuda.driver-version.minor":    driverMinor,
+		"nvidia.com/cuda.driver-version.revision": driverRev,
+		"nvidia.com/cuda.driver-version.full":     driverVersion,
```

Comment:
Question: Do we want to generate a cannonical version string here?

Reply:
What is difference between the canonical version and the ful version string here?

Reply:
https://pkg.go.dev/golang.org/x/mod/semver#Canonical

A Canonical has the .MINOR and .PATCH (revision) included in the string -- that is to say it always consists of three components.

Reply:
So If I understand correctly, aren't the full version and canonical versions the same thing here?

Do we get semver strings that include a build metadata component from the NVML API?

Reply:
No, we don't get that information from the API. We only get major, minor, and patch versions.